Foundation/NSTimeZone.swift

```diff
@@ -249,3 +292,7 @@ extension NSTimeZone {
     }
 
 }
+
+extension NSNotification.Name {
+    public static let NSSystemTimeZoneDidChange = NSNotification.Name(rawValue: "NSSystemTimeZoneDidChangeNotification") // NSUnimplemented
```

Comment:
@xwu No problem. I'm just going to let this CI pass run first before pushing the changes.